apis/Google.Cloud.Firestore.Data/Google.Cloud.Firestore.Data.Tests/CollectionInitializerExtensions.cs

```diff
@@ -0,0 +1,31 @@
+﻿// Copyright 2017, Google Inc. All rights reserved.
```

Comment:
Perhaps not the right place to ask (but I will anyway): now that [Google Inc is Google LLC](https://techcrunch.com/2017/09/04/google-parent-alphabet-forms-holding-company-xxvi-to-complete-2015-corporate-reorganization/), do we need to change these?